electrolyzer/inputs/modeling_schema.yaml

```diff
@@ -87,10 +87,36 @@ properties:
                         description: Number of electrolyzer Stacks to run in simulation
                     control_type:
                         type: string
-                        enum: [PowerSharingRotation, SequentialRotation, EvenSplitEagerDeg, EvenSplitHesitantDeg, SequentialEvenWearDeg, SequentialSingleWearDeg, BaselineDeg]
+                        enum: [PowerSharingRotation, SequentialRotation, EvenSplitEagerDeg, EvenSplitHesitantDeg, SequentialEvenWearDeg, SequentialSingleWearDeg, BaselineDeg, Decision_ctrl]
```

Comment:
this is just a nitpick, but consider using CamelCase to match other names, i.e. `DecisionCtrl`